README.md

```diff
@@ -1,8 +1,75 @@
-# Laminar
+# Laminar | Game Networking Protocol
 
-[![Build Status](https://travis-ci.org/amethyst/laminar.svg?branch=master)](https://travis-ci.org/amethyst/laminar)
+[![Latest Version](https://img.shields.io/crates/v/laminar.svg)](https://crates.io/crates/laminar) | [![Build Status](https://travis-ci.org/amethyst/laminar.svg?branch=master)](https://travis-ci.org/amethyst/laminar) | [![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](./LICENSE) | [![docs.rs](https://docs.rs/laminar/badge.svg)](https://docs.rs/laminar/) | [Examples](https://github.com/amethyst/laminar/tree/master/examples) | [Changelog](LINK) | [Release Nodes](LINK)
```

Comment:
I dont really like this table format tbh.

Reply:
You don't like the table format or you don't like all those links behind each other in one line? I think the table looks nice. And if you don't like the table do you have an example of how you would like it?

Reply:
@TimonPost Im not a big fan of the table format, I think it looks a bit ugly 😄 . Personally, I would just have them listed. Since, we are in the `amethyst` org we should follow how its done on the main repo. https://github.com/amethyst/amethyst